### DISCUSSION TITLE
有无更智能的字幕文件识别方法

### DISCUSSION THREAD

**Discussion by letterk** [hooke007/mpv_PlayKit#132] [Q&A 问答]
目前是需要重命名字幕成电影相同的文件名
单个电影倒是好处理
美剧什么重命名起来就很烦了

**Comment by hooke007** [comment]
我没有这方面的需求，本地做剧集管理也是批量重命名处理。

**Reply by sqq135** [reply]
请问一下是否有字幕的模糊匹配，因为很多电影的视频文件后面会加很长的后缀，和另外来源的字幕后缀不同无法匹配，需要额外修改（不费事，就是看完想收藏，还是希望能保留后缀），如果能有模糊匹配字幕的插件就好很多，或者是快捷键加载同路径下的字幕文件？希望能指路给个链接或者关键词

**Reply by hooke007** [reply]
你说的后缀是什么

**Reply by sqq135** [reply]
类似这样BluRay.1080p.DTS-HD.MA.5.1.x265.10bit-DreamHD.
是不是加上这样的后缀会影响字幕的自动匹配，即使设置sub-auto = fuzzy也不行。

**Reply by hooke007** [reply]
这种直接字幕选择器里手动加载吧，也就点两下鼠标的事情

**Reply by sqq135** [reply]
好的好的
